src/Fixer/Phpdoc/PhpdocOrderByValueFixer.php

```diff
@@ -202,7 +202,7 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedValues([
                     new AllowedValueSubset($allowedValues),
                 ])
-                ->setNormalizer(function (Options $options, $value) {
+                ->setNormalizer(static function (Options $options, $value): array {
```

Comment:
```suggestion
                ->setNormalizer(static function (Options $options, string $value): array {
```

Reply:
I don't think `value` is a `string` here